addons/sales_team/models/res_users.py

```diff
@@ -33,3 +33,7 @@ def _compute_sale_team_id(self):
             else:
                 sorted_memberships = user.crm_team_member_ids  # sorted by create date
                 user.sale_team_id = sorted_memberships[0].crm_team_id if sorted_memberships else False
+
+    def action_archive(self):
+        self.env['crm.team.member'].search([('user_id', 'in', self.ids)]).action_archive()
```

Comment:
Isn't that just `self.crm_team_member_ids`?

Reply:
Ah indeed, could have been a good idea :) anyway does not change much for a small action like this one